hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -499,10 +516,10 @@ public HapiSpec supplyMgmtFailureCasesWork() {
                         burnToken(RIGID, 1).signedBy(GENESIS).hasKnownStatus(TOKEN_HAS_NO_SUPPLY_KEY),
                         mintToken(SUPPLE, Long.MAX_VALUE).hasKnownStatus(INVALID_TOKEN_MINT_AMOUNT),
                         mintToken(SUPPLE, 0).hasPrecheck(OK),
-                        mintToken(SUPPLE, -1).hasPrecheck(INVALID_TOKEN_MINT_AMOUNT),
+                        mintToken(SUPPLE, -1).hasKnownStatus(INVALID_TOKEN_MINT_AMOUNT),
                         burnToken(SUPPLE, 2).hasKnownStatus(INVALID_TOKEN_BURN_AMOUNT),
                         burnToken(SUPPLE, 0).hasPrecheck(OK),
-                        burnToken(SUPPLE, -1).hasPrecheck(INVALID_TOKEN_BURN_AMOUNT));
+                        burnToken(SUPPLE, -1).hasKnownStatus(INVALID_TOKEN_BURN_AMOUNT));
```

Comment:
I think these statuses need to be added to the list of special ingest exceptions (`spec.streamlinedIngestChecks` in `spec-default.properties`) instead of changing from `hasPrecheck` to `hasKnownStatus`

Reply:
Done.